packages/cardano-services/test/StakePool/StakePoolHttpService.test.ts

```diff
@@ -15,7 +15,7 @@ import { DbSyncStakePoolProvider, HttpServer, HttpServerConfig, StakePoolHttpSer
 import { Pool } from 'pg';
 import { doServerRequest } from '../util';
 import { getPort } from 'get-port-please';
-import { stakePoolHttpProvider } from '@cardano-sdk/cardano-services-client';
+import { stakePoolHttpProvider } from '../../../cardano-services-client';
```

Comment:
I think we should use @package-name when using modules from different packages and relative paths when using modules from the same package; looks cleaner IMHO; also, we are doing so in other packages, so it's also more consistent, is there any reason why this needed to change?

Reply:
It's also my preference, but I suspect there's an issue with our tsconfig that is preventing referenced imports in tests, as we've already had to refactor some prior to this change.

> we are doing so in other packages

In source code it's fine, but all tests should now have relative path imports.

Reply:
If we think `from '@package'` syntax has advantages we could go back to using path mappings in tsconfig. Should be a separate task though

Reply:
`ADP-2032`